src/odemis/dataio/tiff.py

```diff
@@ -237,6 +254,17 @@ def _readTiffTag(tfile):
         except Exception as e:
             logging.info(f"Failed to parse ThermoFisher metadata: {e}")
 
+    # parse zeiss metadata
+    zmd = tfile.GetField(TIFFTAG_ZEISS_MD_1)
```

Comment:
Could you add a `ignore_undefined_tag=True` to the call, so that if it's not found (which is very common), it will not generate a warning. (Granted, now the warnings are not shown, but if we ever show them again, it's nice to know that this tag missing is really nothing to worry about). You could also do the same for the TFS tag.